compiler-rt/test/fuzzer/merge-posix.test

```diff
@@ -14,7 +14,7 @@ RUN: echo ....U. > %tmp/T2/2
 RUN: echo ...Z.. > %tmp/T2/3
 RUN: echo ...Z.. > %tmp/T2/4
 RUN: echo ....E. > %tmp/T2/5
-RUN: echo .....R > %tmp/T2/6
+RUN: (echo .....R; for i in {1..1024}; do echo -n X; done) > %tmp/T2/6
```

Comment:
fwiw, before I made this change I did look and found other tests doing similar shell scripting https://github.com/llvm/llvm-project/blob/49d5bb0ad0cb31410184c462801c5049ad671517/compiler-rt/test/asan/TestCases/Darwin/duplicate_os_log_reports.cpp#L13

I'm definitely not sure that this is the best / most portable way to write out a 1024B file 😄. We could just call into `python3` (other tests do that), or use `dd` (but i didn't see any other tests doing that).